### DISCUSSION TITLE
Can we save the SVG generated from rendering a PDF page?

### DISCUSSION THREAD

**Discussion by rshov** [wojtekmaj/react-pdf#933] [Q&A]
Is it possible to get the SVG data that is generated when a PDF page is rendered, so that I can save the SVG to be used later? Thanks!

**Comment by wojtekmaj** [comment]
When an SVG is rendered, it becomes a part of your page. So yes, you can totally do that, by finding `svg` element in your DOM tree and copying its [`outerHTML`](https://developer.mozilla.org/en-US/docs/Web/API/Element/outerHTML).
